examples/Leptonica/CMakeLists.txt

```diff
@@ -0,0 +1,19 @@
+cmake_minimum_required(VERSION 3.0)
+
+# Emulate HunterGate:
+# * https://github.com/hunter-packages/gate
+include("../common.cmake")
+
+project(download-Leptonica)
+
+# download Leptonica
+hunter_add_package(Leptonica)
+find_package(Leptonica CONFIG REQUIRED)
+add_executable(example example.c)
+
+include_directories(${Leptonica_INCLUDE_DIRS})
+target_link_libraries(example ${Leptonica_LIBRARIES})
```

Comment:
That target is never created so that won't work.

> CMake Error at CMakeLists.txt:12 (add_executable):
>   Target "example" links to target "Leptonica::Leptonica" but the target was
>   not found.  Perhaps a find_package() call is missing for an IMPORTED
>   target, or an ALIAS target is missing?

Should I be doing something special to create that target on the Leptonica side?